core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/Resource.java

```diff
@@ -41,6 +41,7 @@
 		public static final String LANGUAGE = ResourceDocument.Fields.LANGUAGE;
 		public static final String CREATED_AT = ResourceDocument.Fields.CREATED_AT;
 		public static final String RESOURCE_TYPE = ResourceDocument.Fields.RESOURCE_TYPE;
+		public static final String TYPE_SORT = ResourceDocument.Fields.TYPE_SORT;
```

Comment:
Rename `typeSort` sortkey to `typeRank`.

Reply:
Done